content/en/docs/Contribute/blog-guidelines.md

```diff
@@ -0,0 +1,94 @@
+<!--
+---
+title: "Create a blog post"
+linkTitle: "Create a blog post"
+weight: 6
+description: >
+  Contributions to the Tekton blog are welcome!
+---
+-->
+
+If you have an interesting, fun, or practical idea that you want to share with the
+world, you can write a blog post about it!
+
+Blog posts don't have to follow strict writing rules or styles. They are a good
+place for you to have fun talking about Tekton.
+
+This is how:
+
+## Set up a local version of the website
+
+1.  [Fork][] the [documentation repository][docs-repo].
+
+1.  Clone the repository to your computer:
+
+    ```bash
+    git clone https://github.com/<your-git-username>/website.git
+  
+    ```
+
+1.  [Install Docker Compose][docker-compose], if it's not already installed on
+    your computer. 
+
+1.  Change directory to the repository root and run the website server:
+
+    ```bash
+    cd website
+    docker-compose up
+    ```
+    And wait for the confirmation message to pop up:
+
+    ```
+    ┌─────────────────────────────────────────────────┐
+    │                                                 │
+    │   ◈ Server now ready on http://localhost:8888   │
+    │                                                 │
+    └─────────────────────────────────────────────────┘
+    ```
+
+    Keep the server running while you write your post. Press *Ctrl + C* to stop
+    it.
+
+1.  Open your browser and go to `http://localhost:8888`.
+
+## Draft your blog post
+
+1.  Create a folder under `content/en/blog/<year>/` with a descriptive name
```

Comment:
Any issues with whitespace or special characters that users need to be aware of?

Reply:
Everything should work, but for consistency with the rest of the guidelines, I added the usual "Lower case, use dashes to separate words".

Reply:
Sounds good, thanks.